packages/packages.csv

```diff
@@ -173,6 +171,7 @@ Microsoft.Extensions.TaskCache.Sources,noship,exclude
 Microsoft.Extensions.TypeNameHelper.Sources,noship,exclude
 Microsoft.Extensions.WebEncoders,ship,include
 Microsoft.Extensions.WebEncoders.Sources,noship,exclude
+Microsoft.Extensions.WebSockets.Internal,noship,exclude
```

Comment:
We have another .Internal package that ships I believe.  I'd like @anurse to comment because he knows the latest on this.